log/mcap.ksy

```diff
@@ -0,0 +1,397 @@
+meta:
+  title: MCAP
+  id: mcap
+  file-extension: mcap
+  license: Apache-2.0
+  endian: le
+doc: |
+  MCAP is a modular container format and logging library for pub/sub messages with
+  arbitrary message serialization. It is primarily intended for use in robotics
+  applications, and works well under various workloads, resource constraints, and
+  durability requirements.
+doc-ref: https://github.com/foxglove/mcap
+seq:
+  - id: header_magic
+    type: magic
+
+  - id: records
+    type: record
+    repeat: until
+    repeat-until: _.op == opcode::footer
+
+  - id: footer_magic
+    type: magic
+
+instances:
+  footer:
```

Comment:
Why does this instance exist if it's not used from within the spec? Isn't the footer record already available in `records`?

Reply:
It is accessible from `records`, however this instance allows direct access via the end of a file (which is a common access pattern for readers that want to use index data — they would not linearly scan all the records in the file from the beginning).

If there were some corruption earlier in the file, this might aid in debugging if the file suffix is intact. At least that was my thinking for including it.

Given that background info, do you think it's better to leave this in or just remove it?